README.md

```diff
@@ -59,6 +59,27 @@ Encountering an issue? Before reaching out to support, we recommend heading to o
 
 If you were using the v1 and wish to update to v2, please follow our [Upgrade Guide](upgrade_guide.md)
 
+## Use app in Docker
+
+If you wish to contribute but would like to avoid installing the repository locally, we provided you with a Docker image. You can build it with the following command
+```.env
+docker build -t algolia-ruby --build-arg ALGOLIA_APPLICATION_ID_1 \
+--build-arg ALGOLIA_ADMIN_KEY_1 \
+--build-arg ALGOLIA_SEARCH_KEY_1 \
+--build-arg ALGOLIA_APPLICATION_ID_2 \
+--build-arg ALGOLIA_ADMIN_KEY_2 \
+--build-arg ALGOLIA_APPLICATION_ID_MCM \
+--build-arg ALGOLIA_ADMIN_KEY_MCM .
```

Comment:
Maybe we could set (example) values in this snippet like:
```
--build-arg ALGOLIA_APPLICATION_ID_1=XXXXXX \
...
```

Also, I think we should **avoid passing secrets as build arguments** (because they will end up in clear in the image). We have two options:
* Passing the secrets as environment variables when running `docker run`
* Using Docker secrets

Reply:
Good point, although as this image will only be used locally I decided that the risks were low enough to put them as args and have a shorten `docker run` command. But extra security doesn't hurt. I didn't know about Docker secrets, I will look into it!

Reply:
@GuiTeK it's updated!